tests/foreman/cli/test_import.py

```diff
@@ -151,6 +167,63 @@ def import_user_data():
     ],)
 
 
+def import_hc_data():
+    """Random data for Organization Import tests"""
+
+    random_ids = random.sample(range(1, 1000), 3)
+    return (
+        {
+            u'orgs': [{
+                u'key_id': u'{}'.format(str(i+1)),
+                u'organization_id': u'{}'.format(str(random_ids[i])),
+                u'organization': gen_string('alphanumeric'),
+            } for i in range(3)],
+            u'hcs': [{
+                u'key_id': u'{}'.format(str(i+1)),
+                u'org_id': u'{}'.format(str(random_ids[i])),
+                u'name': gen_string('alphanumeric'),
+            } for i in range(3)]
+
+        },
+    )
+
+
+def import_repo_data():
```

Comment:
Maybe these functions could be named like gen_import_repo_data or gen_repo_data. Reading import seems that you are importing the repo data and not generating it.